src/SFA.DAS.AdminService.Web/Startup.cs

```diff
@@ -50,14 +50,13 @@ namespace SFA.DAS.AdminService.Web
 {
     public class Startup
     {
-        private readonly IHostingEnvironment _env;
+        private readonly IHostEnvironment _env;
         private readonly ILogger<Startup> _logger;
-        private const string ServiceName = "SFA.DAS.AdminService";
-        private const string Version = "1.0";
+
         public IConfiguration Configuration { get; }
         public IWebConfiguration ApplicationConfiguration { get; set; }
 
-        public Startup(IConfiguration configuration, IHostingEnvironment env, ILogger<Startup> logger)
+        public Startup(IConfiguration configuration, IHostEnvironment env, ILogger<Startup> logger)
         {
             _env = env;
             _logger = logger;
```

Comment:
#if DEBUG
            if (!configuration.IsDev())
            {
                config.AddJsonFile("appsettings.json", false)
                    .AddJsonFile("appsettings.Development.json", true);
            }
#endif

This section is causing some confusion - I think it does not need to explicitly state the appSettings.json should be read as it is always read by default - it being read by default even though this read is DEBUG only is why the SFA.DAS.Provider.DfeSignIn is actually being read currently and the application works.

Reply:
Ok. Amended.